README.md

```diff
@@ -263,26 +263,18 @@ Source code for all the REST resources can be found from package com.spotify.rea
     Delete all the related repair runs before calling this endpoint.
 
 
-Doing a Release (on this repository)
+Doing a Release
 ------------------------------------
 
 1. Go to the root of the cassandra-reaper project on your local Git clone.
-2. In the pom.xml file, remove "-SNAPSHOT" from the version.
-3. Add a new entry to the top of debian/changelog. You may just copy the latest entry and edit (be careful with formatting, and make sure to get the right version number).
-4. Run "mvn clean verify".
-5. Commit the changes with message "release version X.Y.Z".
-6. Make a new tag called vX.Y.Z. Example: v0.1.1.
-7. Run "git push".
-8. Run "git push --tags".
-9. Make a (release) build of the package, if required. For example start a Jenkins build deploy.
-10. In the pom file, increment the version number and append "-SNAPSHOT".
-11. Commit the changes with message "start development on X.Y.Z", where the version matches your current snapshot version.
-12. Run "git push".
-13. Deploy the released packages (there's cassandra-reaper and cassandra-reaper-cli).
-14. Send a release note to users.
-
-Notice that you can get a short list of changes since last release, by e.g.:
-
-```
-$ git log --no-merges --format="%h  %<(18)%an %s" v0.2.1..HEAD
-```
+2. Run `mvn clean verify`.
+3. Commit the changes with message "release version X.Y.Z".
+4. Run `git tag -am X.Y.Z X.Y.Z`. Example: `git tag -am 0.3.0 0.3.0`.
```

Comment:
Are these instructions for if you want to make changes and release or are they for if you want to just get this project and release?  It seems intertwined to me.